src/data/pages/users.ts

```diff
@@ -0,0 +1,190 @@
+import { sleep } from '../../services/utils'
+import { repeatArray } from '../../services/utils'
+import { User } from './../../pages/users/types'
+
+export const users = repeatArray<User>(50, [
+  {
+    id: 1,
+    fullname: 'Patrik Radkow',
+    email: 'magicpan@example.gg',
+    username: 'magicpan',
+    role: 'user',
+    projects: 3,
+    avatar: '',
+    active: true,
+    notes: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, voluptatum.',
+  },
+  {
+    id: 2,
+    fullname: 'Martin Hoff',
+    email: 'niceadmin@mail.com',
+    username: 'admin',
+    role: 'admin',
+    projects: 5,
+    avatar: '😍',
+    active: true,
+    notes: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, voluptatum.',
+  },
+  {
+    id: 3,
+    fullname: 'Liz Macintosh',
+    email: 'ebrown@gmail.com',
+    username: 'ebrown',
+    role: 'user',
+    projects: 2,
+    avatar: 'https://randomuser.me/api/portraits/men/1.jpg',
+    active: true,
+    notes: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, voluptatum.',
+  },
+  {
+    id: 4,
+    fullname: 'M2',
+    email: 'mrm@gmail.com',
+    username: 'mrm',
+    role: 'owner',
+    projects: 1,
+    avatar: '',
+    active: true,
+    notes: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, voluptatum.',
+  },
+  {
+    id: 5,
+    fullname: 'Kevin Smith',
+    email: 'kevin@gmail.com',
+    username: 'kevin13',
+    role: 'user',
+    projects: 13,
+    avatar: 'https://randomuser.me/api/portraits/men/2.jpg',
+    active: true,
+    notes: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, voluptatum.',
+  },
+  {
+    id: 6,
+    fullname: 'Martin Hoff',
+    email: 'martin@gmail.com',
+    username: 'martin3',
+    role: 'user',
+    projects: 2,
+    avatar: 'https://randomuser.me/api/portraits/men/3.jpg',
+    active: true,
+    notes: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, voluptatum.',
+  },
+  {
+    id: 7,
```

Comment:
Let's add some more variety, maybe 10 users on repeat?

![image](https://github.com/epicmaxco/vuestic-admin/assets/5394573/bf237c41-e468-4980-8b19-6710afa0edb1)

Also maybe only nice images, letters are placeholders and we don't have to cover all cases.

Reply:
Almost there. Images are great, but when you open the page you see this:

![image](https://github.com/epicmaxco/vuestic-admin/assets/5394573/0beb3b74-df73-4ea1-a6f1-31bd93dd1f92)

It would be great to see this instead:

![image](https://github.com/epicmaxco/vuestic-admin/assets/5394573/f93ff841-8565-4a71-a646-950f68e62846)

Can we maybe remove John Doe, Martin Hoff etc users without images?